### ISSUE TITLE
can't install by npm 5.4.0 with error below

### ISSUE THREAD

**Issue by kevinaskin** [giggio/node-chromedriver#120]
```
> chromedriver@2.32.2 install /Users/kevinaskin/WebstormProjects/kevinaskin/live/public/node_modules/chromedriver
> node install.js

Downloading https://npm.taobao.org/mirrors/chromedriver/2.32/chromedriver_mac64.zip
Saving to /var/folders/6c/mz_lxxxj0kxbqxkgd910tnsr0000gn/T/chromedriver/chromedriver_mac64.zip
ChromeDriver installation failed Error with https: request: Error: Protocol "https:" not supported. Expected "http:"
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chromedriver@2.32.2 install: `node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the chromedriver@2.32.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
```

and `.npmrc` includes: 
```
chromedriver_cdnurl = "https://npm.taobao.org/mirrors/chromedriver"
strict-ssl = false
```

PS:
* when I use `cnpm install`, it is okay to install successfully
* I check this path `/var/folders/6c/mz_lxxxj0kxbqxkgd910tnsr0000gn/T/chromedriver/chromedriver_mac64.zip`,
this file is exists already

**Comment by giggio** [comment]
Bug confirmed, also failed on Windows.

**Comment by giggio** [comment]
Published to npm as `2.32.3`. Thanks for reporting it.
